corehq/apps/app_manager/util.py

```diff
@@ -409,7 +409,7 @@ def get_casedb_schema(form):
     This lists all case types and their properties for the given app.
     """
     app = form.get_app()
-    base_case_type = form.get_module().case_type
+    base_case_type = form.get_module().case_type if form.requires_case() else None
```

Comment:
Any reason to not make this return from the data dictionary if it exists? I think that would be the preferred way going forward